### ISSUE TITLE
[Learn/Dashboard] Chapter 11 to Chapter 12, code discontinuity

### ISSUE THREAD

**Issue by Connor-Skudlarek** [vercel/next-learn#434]
This is accurate to 8:53pm PST for the dashboard app project using Next.js 14. Sorry if the markup is hard to follow, I'm trying to highlight the issue.

The file path:

`app/ui/invoices/table.tsx`

In chapter 11, this function looks like so:

export default async function InvoicesTable({

  
```
  query,
  currentPage,
```

}: {
 ```
  query: string;
  currentPage: number;
```
}) {
  const invoices = await fetchFilteredInvoices(query, currentPage);
}

The parameters are "query" and "currentPage".

But in chapter 12, it's like this:

export default async function InvoicesTable({

`  invoices,`

}: {

`  invoices: InvoiceTable[];
`
}) {
  return (
   ...
  );
}

Now the parameter is "invoices" drawing on an array following the type "InvoiceTable". But this is never altered between the two chapters.

**Comment by delbaoliveira** [comment]
Good catch @Connor-Skudlarek, thank you 🙏🏼 Updated, should be landing soon.
